Web/Assignmrnt 2/website.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
```

Comment:
ACTION: checking this on "http://jsfiddle.net/" i don't find this looking what was expected. The site title and footer is not covering the whole width.